### PR TITLE
[Feature] County Insights Card + Section

### PR DESCRIPTION
**Pull request by gertie-sheshe**
## Description

Add Data Driven Insights Card and Section

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshots
Screenshot 2021-07-26 at 10 34 02
Screenshot 2021-07-23 at 03 35 59
Screenshot 2021-07-23 at 03 36 10
Screenshot 2021-07-23 at 03 36 21

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/components/InsightsData/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      <Typography
+        display="inline"
+        className={classes.subtitle}
+      >{`${subTitleFirst} `}</Typography>
+      <Typography
+        display="inline"
+        className={clsx(classes.subtitle, classes.subtitleTwo)}
+      >
+        {subTitleEnd}
+      </Typography>
```
[the remark this answers is not in the document]
Comment (gertie-sheshe) [reply src/components/InsightsData/index.js]: Fixed

**Review by nerdlet — APPROVED** [review]
@gertie-sheshe Looks good! Quick one for me would be the `readme` text should be 14px 👇🏽 
Screenshot 2021-07-29 at 11 03 14

File: src/components/InsightsData/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  const classes = useStyles(props);
+
+  return (
+    <Section classes={{ root: classes.section }}>
+      <Typography className={classes.title}>{title}</Typography>
+      <RichTypography className={classes.subtitle}>{subtitle}</RichTypography>
+      <div className={classes.list}>
+        {itemsData &&
+          itemsData.map((item) => <InsightCard key={item.title} {...item} />)}
+      </div>
```
Comment (KhadijaMahanga) [inline src/components/InsightsData/index.js:27]: Grid container, Grid item should be your friend

File: src/components/InsightsData/index.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+};
+
+InsightsData.propTypes = {
+  title: PropTypes.string,
+  subtitle: PropTypes.string,
+  items: PropTypes.arrayOf(
+    PropTypes.shape({
+      title: PropTypes.string,
+      description: PropTypes.string,
+      image: PropTypes.objectOf(PropTypes.any),
```
Comment (KhadijaMahanga) [inline src/components/InsightsData/index.js:39]: `objectOf` !== `string`

File: src/functions/formatBlocksForSections.js [hunk 4]
```diff
@@ -23,6 +23,8 @@ function format(block) {
   switch (name) {
     case "lazyblock/explore-other-tools":
       return formatLazyBlockIteratorContentWithImage(attributes, "image");
+    case "lazyblock/data-insights":
+      return formatLazyBlockIteratorContentWithImage(attributes, "image");
```
Comment (KhadijaMahanga) [inline src/functions/formatBlocksForSections.js:27]:
```suggestion
    case "lazyblock/data-insights":
    case "lazyblock/explore-other-tools":
      return formatLazyBlockIteratorContentWithImage(attributes, "image");
```

File: src/components/InsightsData/useStyles.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+const useStyles = makeStyles(({ typography, breakpoints }) => ({
+  root: {},
+  section: {
+    paddingTop: typography.pxToRem(60),
+    "& .highlight": {
+      background:
+        "linear-gradient(180deg,rgba(255,255,255,0) 30%, #0067A31A 50% )",
+      display: "inline-block",
+    },
```
Comment (KhadijaMahanga) [inline src/components/InsightsData/useStyles.js:11]: Why have this on section instead of subtitle?

File: src/components/InsightsData/index.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+import useStyles from "./useStyles";
+
+import InsightCard from "@/pesayetu/components/InsightCard";
+import Section from "@/pesayetu/components/Section";
+
+const InsightsData = ({ title, subtitle, items, ...props }) => {
+  let itemsData = [];
+  const theme = useTheme();
+  const isTablet = useMediaQuery(theme.breakpoints.only("md"));
+
```
Comment (KhadijaMahanga) [inline src/components/InsightsData/index.js:16]: Let's use `Hidden`

**Review by KhadijaMahanga — COMMENTED** [review]
@gertie-sheshe most of visuals PR feedback have repeated in this one too

File: src/components/InsightsData/index.js (hunk 6 above)
Comment (gertie-sheshe) [reply src/components/InsightsData/index.js:16]: @KhadijaMahanga this is just to determine how many items to be rendered and not to render a different design.

File: src/components/InsightsData/useStyles.js (hunk 5 above)
Comment (gertie-sheshe) [reply src/components/InsightsData/useStyles.js]: Fixed

File: src/components/InsightsData/index.js (hunk 3 above)
Comment (gertie-sheshe) [reply src/components/InsightsData/index.js]: Lol, thanks. Fixed

File: src/functions/formatBlocksForSections.js (hunk 4 above)
Comment (gertie-sheshe) [reply src/functions/formatBlocksForSections.js]: Fixed

**Review by KobbyMmo — APPROVED** [review]
Looks good to me @gertie-sheshe

**Pull request merged**